hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -262,7 +262,8 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       Map<String, String> summary = Optional.ofNullable(metadata.currentSnapshot())
           .map(Snapshot::summary)
           .orElseGet(ImmutableMap::of);
-      setHmsTableParameters(newMetadataLocation, tbl, metadata.properties(), removedProps, hiveEngineEnabled, summary);
+      setHmsTableParameters(newMetadataLocation, tbl, metadata.properties(), removedProps, hiveEngineEnabled, summary,
```

Comment:
I wonder if we should be passing through the metadata here instead of `"tbl, metadata.properties and uuid"`

Reply:
Made the change